infrastructure/resource_group_level/modules/privatelink-azurewebsites-net.bicep

```diff
@@ -0,0 +1,521 @@
+param azureWebsitesDnsZoneName string = 'privatelink.azurewebsites.net'
+param vnetId string
+
+resource azureWebsitesDnsZone 'Microsoft.Network/privateDnsZones@2018-09-01' = {
+  name: azureWebsitesDnsZoneName
+  location: 'global'
+  tags: {
+    Environment: 'Preproduction'
+  }
+}
+
+resource azureDnsSoaRecord 'Microsoft.Network/privateDnsZones/SOA@2018-09-01' = {
+  parent: azureWebsitesDnsZone
+  name: '@'
+  properties: {
+    ttl: 3600
+    soaRecord: {
+      email: 'azureprivatedns-host.microsoft.com'
+      expireTime: 2419200
+      host: 'azureprivatedns.net'
+      minimumTtl: 10
+      refreshTime: 3600
+      retryTime: 300
+      serialNumber: 1
+    }
+  }
+}
+
+resource appServiceVnetLink 'Microsoft.Network/privateDnsZones/virtualNetworkLinks@2018-09-01' = {
+  parent: azureWebsitesDnsZone
+  name: 'app-service-dns'
+  location: 'global'
+  tags: {
+    Environment: 'Preproduction'
+  }
+  properties: {
+    registrationEnabled: false
+    virtualNetwork: {
+      id: vnetId
+    }
+  }
+}
+
+// TODO:DTFS-6422 Wire up A record IPs correctly, getting appropriate values.
```

Comment:
I need to understand how the IPs are allocated before defining them. I certainly can't just use the values already there as they are from Dev (and "Demo"). I suspect I'll uncomment (or even remove if they are more appropriately created elsewhere) IPs as I add other modules.